content/300-guides/300-upgrade-guides/200-upgrading-versions/600-upgrading-to-prisma-5.mdx

```diff
@@ -0,0 +1,321 @@
+
+</Admonition>
+
+### PostgreSQL minimum version change
+
+From Prisma version 5.0.0, the minimum version of PostgreSQL supported is version 9.6. If your project uses an earlier version of PostgreSQL, you will need to update it.
+
+Our tests have not been testing PostgreSQL 9.4 and 9.5 for some time, so we are formally removing support.
+
+### Prisma Client embedded SQLite version updated
+
+With Prisma version 5.0.0, we have upgraded the embedded version of SQLite from `3.35.4` to `3.41.2`. We did not see any breaking changes and don't anticipate any changes needed in user projects, but if you are using SQLite, make sure to check [the SQLite changelog](https://www.sqlite.org/changes.html)
+
+### Removal of array shortcuts
+
+Prisma version 5.0.0 drops support for array shortcuts, which were a way to omit a wrapping array for single values.
+
+#### `OR` clauses
+
+The following code in Prisma 4:
+
+```js
+prisma.user.findMany({
+  OR: { email: 'foo@example.com' },
+})
+```
+
+Will need to be changed to the following in Prisma 5:
+
+```js highlight=2;normal
+prisma.user.findMany({
+  OR: [{ email: 'foo@example.com' }],
+})
+```
+
+`OR` clauses will only accept array values.
+
+#### `in` and `notIn` clauses
+
+Similar to `OR`, `in` and `notIn` require array values.
+
+Prisma 4:
+
+```js
+prisma.user.findMany({
+  id: { in: 123 },
+})
+
+prisma.user.findMany({
+  id: { notIn: 123 },
+})
+```
+
+Prisma 5:
+
+```js highlight=2,6;normal
+prisma.user.findMany({
+  id: { in: [123] },
+})
+
+prisma.user.findMany({
+  id: { notIn: [123] },
+})
+```
+
+If your `in` and `notIn` values are only one element, you can also update your code to not use these clauses:
+
+```js highlight=2,6;normal
+prisma.user.findMany({
+  id: 123,
+})
+
+prisma.user.findMany({
+  id: { not: 123 },
+})
+```
+
+#### `path` argument for filtering on JSON fields in Postgres
+
+[When filtering on JSON fields in a Postgres table](https://www.prisma.io/docs/concepts/components/prisma-client/working-with-fields/working-with-json-fields#filter-on-a-json-field) the `path` argument now only accepts an array.
+
+Prisma 4:
+
+```prisma
+datasource db {
+  provider = "postgresql"
+}
+
+model User {
+  id       String @id
+  settings Json
+}
+```
+
+```js
+prisma.user.findMany({
+  where: {
+    settings: {
+      path: 'someSetting',
+      equals: someValue,
+    },
+  },
+})
+```
+
+Prisma 5
+
+```prisma
+datasource db {
+  provider = "postgresql"
+}
+
+model User {
+  id       String @id
+  settings Json
+}
+```
+
+```js highlight=4;normal
+prisma.user.findMany({
+  where: {
+    settings: {
+      path: ['someSetting'],
+      equals: someValue,
+    },
+  },
+})
+```
+
+<Admonition type="info">
+
+Note: This change only affects PostgreSQL databases. MySQL databases are not affected.
+
+</Admonition>
+
+#### Composite and Scalar lists
+
+Inputs for operations on Composite and Scalar lists now only accept array values. The following examples all use the following schema:
```

Comment:
"Composite lists" and "Scalar lists" could use a link here. Our terminology is not the most common ones, so giving people to figure out what it is might be a good idea.